apple/Sources/FerrostarSwiftUI/Views/SpeedLimit/USSpeedLimitView.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+
+public struct USSpeedLimitView: View {
```

Comment:
I struggled to find the most correct name for this, but it's used outside the US too, so I think we can just call it "US-style" for now.

```suggestion
/// A US-style speed limit sign using a rectangular white box and a black border.
public struct USStyleSpeedLimitView: View {
```